movo_demos/scripts/voice_nav.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+"""
+This code is meant to control MOVO with voice commands using a map created prior to the demo. In short, when I tell it to go somewhere, for example Nassif's office, it should go there.
+"""
+
+from tf.transformations import euler_from_quaternion, quaternion_from_euler
+import rospy
+from geometry_msgs.msg import Pose, PoseStamped, PointStamped, PoseWithCovarianceStamped, Point, Quaternion, Twist
+import os
+from std_msgs.msg import String, Bool
+#The goals are predetermined positions in a map with a location name as key. They need to be changed to work with a specific map.
+
+goals = { "boss": [-3.86, 5.36, 0.000, 1],"friend": [-3.98, -3.54, 0.000, 1], "hallway":[-4.529,0.1,0.0,1.0], "home": [-0.42,0.1, 0.0,1.0], "lobby":[4.36,-4.96,0,1.0]} #test_zone
+#goals = { "boss": [-5.0, 2.34, 0.000, 0],"friend": [0.04, 2.18, 0.000, -1.385], "hallway":[-6.66,0.18,0.0,1.0], "home": [-2.44,-5.826, 0.0,1.385], "lobby":[-6.8,0.28,0,0]} #soussol2
```

Comment:
are all those commented goals still useful? If no, we should remove them to avoid confusion